webapps/frontend/app/views/site/dashboard-alba.html

```diff
@@ -21,7 +21,7 @@
             <a data-bind="popover: {
                               title: 'alba:dashboard.helptitle',
                               content: 'alba:dashboard.helpcontent'
-                          }" class="hand fa noline" data-i18n="[html]ovs:icons.help"
+                          }" class="hand fa noline" data-i18n="[html]ovs:icons.question_circle"
```

Comment:
In the future, I'd rather see here icons.help again, so it's agnostic of the icon that is used effectively.